Sources/Preferences/util.swift

```diff
@@ -47,3 +47,52 @@ extension NSView {
 		return result
 	}
 }
+
+extension NSWindow {
+	var effectiveMinSize: CGSize {
```

Comment:
@DivineDominion Would you be able to add a doc comment to this? I'm not exactly sure what it does and when/why it's needed.

Reply:
I used this for a while before requiring correct view sizing though AutoLayout, when we still were setting the frame manually. The window would shrink to (0,0) size and be effectively invisible in some cases. Apparently, I forgot to remove it later, so it can be thrown away.